lib/lumberg/format_whm.rb

```diff
@@ -59,7 +61,7 @@ def format_response(response)
         message = response['error']
       when :xfer
         success, message, params = format_xfer_response(response)
-      else
+      when :unknown
```

Comment:
Yes, the `response_type` method already implements a catch-all condition: https://github.com/site5/lumberg/blob/75b9ab8bdd0c24e3667dd6c3d6ac5cb087f77d2c/lib/lumberg/format_whm.rb#L81-L83